astropy/time/tests/test_basic.py

```diff
@@ -616,7 +613,7 @@ def test_bad_time(self):
         djm0 = np.array([0], dtype=np.double)
```

Comment:
Not really this PR's fault, but could you remove these two assignments? `dmj0` and `dmj` get overridden anyway.